src/domain/enrolment/__tests__/CreateEnrolmentPage.test.tsx

```diff
@@ -576,3 +603,126 @@ test('Allow sms notifications if any of the phone numbers are given', async () =
     expect(screen.getByLabelText(/Tekstiviestillä/i)).toBeDisabled();
   });
 });
+
+describe('form local storage', () => {
+  const testValues = {
+    hasEmailNotification: true,
+    hasSmsNotification: false,
+    isSameResponsiblePerson: true,
+    isSharingDataAccepted: false,
+    isMandatoryAdditionalInformationRequired: false,
+    language: '',
+    maxGroupSize: 0,
+    minGroupSize: 0,
+    studyGroup: {
+      person: {
+        name: 'Test person',
+        phoneNumber: '12312343657',
+        emailAddress: 'test@test.com',
+      },
+      name: 'study group name',
+      groupName: 'group name',
+      groupSize: '10',
+      amountOfAdult: '1',
+      studyLevels: [StudyLevel.Grade_1],
```

Comment:
It should be noted that the StudyLevels does not match with the keywords in the TARGET_GROUP keyword set, which is a really sad thing, because prefilling the study level in enrolment and to recommend events related to the study level / target group was the most desired feature of them all.

```
export enum StudyLevel {
  Preschool = 'PRESCHOOL',
  Grade_1 = 'GRADE_1',
  Grade_2 = 'GRADE_2',
  Grade_3 = 'GRADE_3',
  Grade_4 = 'GRADE_4',
  Grade_5 = 'GRADE_5',
  Grade_6 = 'GRADE_6',
  Grade_7 = 'GRADE_7',
  Grade_8 = 'GRADE_8',
  Grade_9 = 'GRADE_9',
  Grade_10 = 'GRADE_10',
  Secondary = 'SECONDARY',
}
```

Keywords in TARGET_GROUP keyword set: https://api.hel.fi/linkedevents/v1/keyword_set/kultus:target_groups/?include=keyword

```
{
    "id": "kultus:target_groups",
    "keywords": [
        {
            "@id": "https://api.hel.fi/linkedevents/v1/keyword/kultus:40/"
        },
        {
            "@id": "https://api.hel.fi/linkedevents/v1/keyword/kultus:51/"
        },
        {
            "@id": "https://api.hel.fi/linkedevents/v1/keyword/kultus:52/"
        },
        {
            "@id": "https://api.hel.fi/linkedevents/v1/keyword/kultus:54/"
        },
        {
            "@id": "https://api.hel.fi/linkedevents/v1/keyword/kultus:55/"
        },
        {
            "@id": "https://api.hel.fi/linkedevents/v1/keyword/kultus:56/"
        },
        {
            "@id": "https://api.hel.fi/linkedevents/v1/keyword/kultus:57/"
        },
        {
            "@id": "https://api.hel.fi/linkedevents/v1/keyword/kultus:58/"
        },
        {
            "@id": "https://api.hel.fi/linkedevents/v1/keyword/kultus:59/"
        },
        {
            "@id": "https://api.hel.fi/linkedevents/v1/keyword/yso:p17354/"
        }
    ],
    //...
}
```

Reply:
There are 10 keywords in the keyword set, but 12 options in study levels 🤔  The keyword set also includes age groups like "0-2 years old", "3-4 years old" and "5-6 years old". After them comes "Preparatory education".

Reply:
hmm 🤔

Reply:
This is something that is hard to map and maintain. The keywords in the LinkedEvents are not maintained by us, and the ids' can vary on different servers. They actually aren't even the same thing, because another one is only a dynamic tag that is linked to an event to tell what kind of event it is, but the another tells the fact about the study group. The mapping could be done in API too, but currently, API does not need it anywhere: The local storage used by UI is the only place that currently needs to map study levels to the keywords. 🤔